script-based-workflows/R/03_histogram.R

```diff
@@ -0,0 +1,21 @@
+# A plot of the resulting munged data will be generated and saved in the data
```

Comment:
These comments at the top of each script are really helpful. Would you use present tense and active voice?

Reply:
Sure thing. It is standard and suggested practice from [Good enough practices in scientific computing](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005510#sec005). I try to follow it in my own code development when I have taken script-based workflows in the past.